src/store/store.ts

```diff
@@ -212,6 +212,38 @@ export class Store {
 		return pageRef ? pageRef.getProject() : undefined;
 	}
 
+	/**
+	 * Returns the page element for a given ID
+	 * @return The page element.
+	 */
+	public getElementById(id: number[], startElement?: PageElement): PageElement | undefined {
+		if (!id) {
```

Comment:
not anymore. This was still based on the string implementation of it

Reply:
Now i remember. This checks if the id is a empty array and then returns early.

Reply:
but then the check would be `if (!id.length)` right?

Reply:
Not really needed. A empty array is falsy :)

Reply:
i think that's a common misconception since empty strings are falsey. you can try it in your browsers console `if([]) console.log('empty but not falsey')`

https://codepen.io/arosenb2/post/is-a-javascript-empty-array-falsey-or-not

Reply:
yeah you are right. This also means this code bit is not necessary.
For me the misconception came because of this:
```
[] == true;
// false

[] == false;
// true
```